common/src/main/java/tachyon/Constants.java

```diff
@@ -284,5 +284,12 @@
   public static final int BYTES_WRITTEN_REMOTE_INDEX = 9;
   public static final int BYTES_WRITTEN_UFS_INDEX = 10;
 
+  /** Security */
+  public static final String TACHYON_SECURITY_USERNAME = "tachyon.security.username";
+  // Authentication
+  public static final String TACHYON_SECURITY_AUTHENTICATION = "tachyon.security.authentication";
+  public static final String TACHYON_AUTHENTICATION_PROVIDER_CUSTOM_CLASS =
```

Comment:
A TODO added. I will make it in a followup PR when everyone is fine with these configuration properties.